test.js

```diff
@@ -10,9 +10,9 @@ test(async t => {
 	t.false(await fn(notClean));
 	t.true(fn.sync(clean));
 	t.false(fn.sync(notClean));
-	t.false(await fn(notClean, {ignore: ['dirty.txt']}));
-	t.true(await fn(notClean, {ignore: ['dirty.txt', 'dirtier.txt']}));
-	t.true(await fn(notClean, {files: ['dirty.txt'], ignore: ['dirty.txt']}));
-	t.true(await fn.sync(notClean, {ignore: ['dirty.txt', 'dirtier.txt']}));
-	t.true(await fn.sync(notClean, {files: ['dirty.txt'], ignore: ['dirty.txt']}));
+	t.false(await fn(notClean, {files: ['!dirty.txt']}));
+	t.true(await fn(notClean, {files: ['!dirty.txt', '!dirtier.txt']}));
+	// t.true(await fn(notClean, {files: ['dirty.txt', '!dirty.txt']}));
+	t.true(await fn.sync(notClean, {files: ['!dirty.txt', '!dirtier.txt']}));
+	// t.true(await fn.sync(notClean, {files: ['dirty.txt', '!dirty.txt']}));
```

Comment:
not sure why those don't work, but it really doesn't make sense to use it that way anyways.